### PR TITLE
Hardcode Remaining Beneficiaries

### PR DESCRIPTION
**Pull request by ug02fast**
Hardcoded beneficiary and vesting addresses.

Dune query: https://dune.xyz/queries/281370

### CODE REVIEW

File: src/pages/VestingClaim.tsx [hunk 1]
```diff
@@ -53,11 +53,28 @@ function VestingClaim(): ReactElement {
         const currentBeneficiaryEvents = events.filter(
           (event) => event.args?.beneficiary === account,
         )
-        const isValidBeneficiary = currentBeneficiaryEvents.length > 0
+        const nonCapturedBeneficiaryAddresses: {
```
Comment (ug02fast) [inline src/pages/VestingClaim.tsx:56]: map of beneficiary address -> vesting contract address

File: src/pages/VestingClaim.tsx [hunk 2]
```diff
@@ -53,11 +53,28 @@ function VestingClaim(): ReactElement {
...
+          [beneficiaryAddress: string]: string
+        } = {
+          "0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE":
```
Comment (ug02fast) [inline src/pages/VestingClaim.tsx:59]: checksummed casing considered

File: src/pages/VestingClaim.tsx [hunk 3]
```diff
@@ -53,11 +53,28 @@ function VestingClaim(): ReactElement {
...
+            "0x5DFbCeea7A5F6556356C7A66d2A43332755D68A5",
+          "0xEC6f7607cD7E4C942a75d40C269deC01BBc9A15e":
+            "0x597e475a5ddd90b3eb2135ac47319bd866f685d8",
+          "0x53ab66E5bAb196CF86F65feD79981cb85470200e":
+            "0x5c85B43468da23F86016f508f14cA927bfD8A737",
+          "0x17f61d0F9701A7fB5814C2d4AD3dC3831e07b277":
+            "0x41092b4ecf2c4db719ec5ab67dbd0c66f095ee97",
+        }
+        const isValidBeneficiary =
```
Comment (ug02fast) [inline src/pages/VestingClaim.tsx:68]: you're a valid beneficiary if you're picked up in `currenctBeneficiaryEvents` or have a value in `nonCapturedBeneficiaryAddresses`

File: src/pages/VestingClaim.tsx [hunk 4]
```diff
@@ -53,11 +53,28 @@ function VestingClaim(): ReactElement {
...
+          currentBeneficiaryEvents.length > 0 ||
+          Boolean(nonCapturedBeneficiaryAddresses[account])
         setIsValidBeneficiary(isValidBeneficiary)
         if (isValidBeneficiary) {
+          const vestingContractAddress =
```
Comment (ug02fast) [inline src/pages/VestingClaim.tsx:73]: if your address isn't in the `nonCapturedBeneficiaryAddresses` object then undefined then you have to be in `currentBeneficiaryEvents`.

**Review by hammeiam — APPROVED** [review]
Approving to unblock, but make sure to update the `nonCapturedBeneficiaryAddresses` with the values from the query that returns the `SetBeneficiary` events.

**Pull request merged**